docs/refmans/gui/viselements/generic/tree.md_template

```diff
@@ -51,32 +51,26 @@ the same level, the first node will be automatically collapsed.
 The [*expanded*](#p-expanded) property can also hold a list of node identifiers that are expanded.
 
 !!! example "Definition"
-
+    === "Python"
+        ```python
+        import taipy.gui.builder as tgb
+        ...
+        tgb.tree("value", lov="Item 1;Item 2;Item 3", expanded=False)
```

Comment:
Why not have lists in lov for Python API in this context?

Reply:
Because it was generated.
But we can add this, yes.